math/minuit2/inc/Minuit2/FCNGradientBase.h

```diff
@@ -38,7 +43,24 @@ class FCNGradientBase : public FCNBase {
 
    virtual std::vector<double> Gradient(const std::vector<double> &) const = 0;
 
+   virtual std::vector<double> G2ndDerivative(const std::vector<double> &) const = 0;
```

Comment:
Do we need them at pure virtual functions ? I would add them as an option, returning a dummy vector in the base class.

Reply:
We don't need them at all! ;) Removed, resolved.